SingularityUI/app/views/taskHealthcheckNotificationSubview.coffee

```diff
@@ -27,6 +29,9 @@ class taskHealthcheckNotificationSubview extends View
         lastHealthcheckFailed: @model.get('healthcheckResults')?.length > 0 and @model.get('healthcheckResults')[0].statusCode isnt 200
         synced:           @model.synced
         config:           config
+        tooManyRetries: @model.get('healthcheckResults').length >= maxRetries
```

Comment:
Small thing, number of retries doesn't include the first try, meaning we have exhausted our retries if there are `maxRetries + 1` failed health checks present. Currently, this may unintentionally come up with exhausted retries when it is actually a timeout.

Related, you are using numberFailed as the number of failed retries in the message, but defining it as the number of failed total. Either way is fine, just needs to be consistent in messaging